synchronization/src/androidTestCyface/java/de/cyface/synchronization/SyncAdapterTest.java

```diff
@@ -38,14 +38,14 @@
 import de.cyface.utils.Validate;
 
 /**
- * Tests the correct internal workings of the <code>CyfaceSyncAdapter</code> with the database.
+ * Tests the correct internal workings of the <code>SyncAdapter</code> with the database.
  *
  * @author Armin Schnabel
  * @version 1.0.0
  * @since 2.4.0
  */
 @RunWith(AndroidJUnit4.class)
```

Comment:
The Syncadapter is no longer cyface-specific and moved out of the flavour folder, thus, the name is changed.